CODEOWNERS

```diff
@@ -153,7 +152,6 @@ models/demos/t3000/llama2_70b @cglagovichTT @caixunshiren @uaydonat
 models/demos/t3000/llama3_70b @cglagovichTT @caixunshiren @uaydonat
 models/demos/t3000/mixtral8x7b @yieldthought @mtairum @uaydonat
 models/demos/grayskull @boris-drazic @eyonland
-models/experimental/
```

Comment:
Wait what why? Now we're back to Boris having to approve everything, no? Or am I wrong